command/cmdfactory.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
No, I added it for something I decided to not add, and forgot to remove the import, didn't feel like making another pull for just that.